src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -14,6 +14,7 @@
             [liberator.core :refer [defresource]]
             ;[liberator.dev :as liberator-dev]
```

Comment:
Haha yeah @KevinCorcoran threw the comment in some liberator resource somewhere after a particularly frustrating week of development, if I remember correctly.

I'd say just start removing these comments from the codebase.  If there's really something to be shared to other developers then we should write formal documentation somewhere, even if it's only a paragraph or two.